content/millennium/r4/medications/medication-request.md

```diff
@@ -139,10 +139,54 @@ We are currently always returning a value of "Order" in the intent field. We rec
 
 The common [errors] and [OperationOutcomes] may be returned.
 
-[`token`]: http://hl7.org/fhir/R4/search.html#token
-[`reference`]: http://hl7.org/fhir/R4/search.html#reference
-[`date`]: http://hl7.org/fhir/R4/search.html#date
-[`number`]: http://hl7.org/fhir/R4/search.html#number
-[status]: https://www.hl7.org/fhir/r4/valueset-medicationrequest-status.html
-[errors]: ../../#client-errors
-[OperationOutcomes]: ../../#operation-outcomes
+## Create
+
+Create an individual MedicationRequest.
+
+    POST /MedicationRequest
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', 'Content-Type': 'application/fhir+json'} %>
+
+### Body Fields
+
+<%= definition_table(:medication_request, :create, :r4) %>
+
+### Example
+
+#### Request
+
+    POST https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/MedicationRequest
+
+#### Body
+
+<%= json(:r4_medication_request_create) %>
+
+#### Response
+
+<%= headers status: 201 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: application/fhir+json
+Date: Wed, 27 Mar 2019 15:59:33 GMT
+Etag: W/"0"
+Last-Modified: Wed, 27 Mar 2019 15:59:30 GMT
+Location: https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/MedicationRequest/4595905
+Server-Response-Time: 3890.363996
+Status: 201 Created
+Vary: Origin
+X-Request-Id: 1638e30e497b93ff4383b2ff0eaeea68
+X-Runtime: 3.890282
+</pre>
+
+The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
```

Comment:
probably leave this note off of resources that don't expose updates?

Reply:
Removed.